docs/cis_self_assessment.md

```diff
@@ -355,225 +351,106 @@ root:root
-For example,
-
-``` bash
-chmod -R 644 /etc/kubernetes/ssl
-```
-
-**Audit Script:** check_files_permissions.sh
-
-```
-#!/usr/bin/env bash
-
-# This script is used to ensure the file permissions are set to 644 or
-# more restrictive for all files in a given directory or a wildcard
-# selection of files
-#
-# inputs:
-#   $1 = /full/path/to/directory or /path/to/fileswithpattern
-#                                   ex: !(*key).pem
-#
-#   $2 (optional) = permission (ex: 600)
-#
-# outputs:
-#   true/false
-
-# Turn on "extended glob" for use of '!' in wildcard
-shopt -s extglob
-
-# Turn off history to avoid surprises when using '!'
-set -H
-
-USER_INPUT=$1
-
-if [[ "${USER_INPUT}" == "" ]]; then
-  echo "false"
-  exit
-fi
-
-
-if [[ -d ${USER_INPUT} ]]; then
-  PATTERN="${USER_INPUT}/*"
-else
-  PATTERN="${USER_INPUT}"
-fi
-
-PERMISSION=""
-if [[ "$2" != "" ]]; then
-  PERMISSION=$2
-fi
-
-FILES_PERMISSIONS=$(stat -c %n\ %a ${PATTERN})
-
-while read -r fileInfo; do
-  p=$(echo ${fileInfo} | cut -d' ' -f2)
-
-  if [[ "${PERMISSION}" != "" ]]; then
-    if [[ "$p" != "${PERMISSION}" ]]; then
-      echo "false"
-      exit
-    fi
-  else
-    if [[ "$p" != "644" && "$p" != "640" && "$p" != "600" ]]; then
-      echo "false"
-      exit
-    fi
-  fi
-done <<< "${FILES_PERMISSIONS}"
-
-
-echo "true"
-exit
-```
-
-**Audit Execution:**
-
-```
-./check_files_permissions.sh '/etc/kubernetes/ssl/\*.pem'
+**Audit:** 
+```bash
+stat -c %n\ %a /var/lib/rancher/rke2/server/tls/*.crt
 ```
-<!-- TODO escaped asterisk -->
-**Expected result**:
 
-```
-'true' is present
+**Remediation:**
+```bash
+chmod -R 644 /var/lib/rancher/rke2/server/tls
 ```
 
 
 #### 1.1.21
 Ensure that the Kubernetes PKI key file permissions are set to `600` (Scored)
 <details>
 <summary>Rationale</summary>
-Kubernetes makes use of a number of key files as part of the operation of its components. The permissions on these files should be set to 600 to protect their integrity and confidentiality.
+Kubernetes makes use of a number of key files as part of the operation of its components. The permissions on these files should be set to 600 to protect their integrity and confidentiality. This is a manual check and each file in the output will need to be verified.
 </details>
 
 **Result:** Pass
 
-**Remediation:**
-Run the below command (based on the file location on your system) on the master node.
-For example,
-
-``` bash
-chmod -R 600 /etc/kubernetes/ssl/certs/serverca
-```
-
-**Audit Script:** 1.1.21.sh
-<!-- TODO escaped asterisk -->
-```
-#!/bin/bash -e
-check_dir=${1:-/etc/kubernetes/ssl}
-
-for file in $(find ${check_dir} -name "\*key.pem"); do
-file_permission=$(stat -c %a ${file})
-if [[ "${file_permission}" == "600" ]]; then
-continue
-else
-echo "FAIL: ${file} ${file_permission}"
-exit 1
-fi
-done
-
-echo "pass"
-```
-
-**Audit Execution:**
-
-```
-./1.1.21.sh /etc/kubernetes/ssl
+**Audit**
+```bash
```

Comment:
Same comment as above, the audit should be a little more verbose, like:
**Audit:**

Run the below command on your master nodes:
```bash
stat -c %n\ %a /var/lib/rancher/rke2/server/tls/*.key
```
Verify that the permissions are 600.